ios-trivia-game/Models/Answer.swift

```diff
@@ -9,30 +9,34 @@
 import Foundation
 
 class Answer: NSObject {
-    var user_id: String!
-    var user_selected_answer_text: String!
-    var correct_answer_text: String!
-    var question_id: String!
-    var timestamp: Date!
-    var room_id: String!
+    var userId: String!
+    var answerText: String!
+    var questionId: Int!
+    var timestamp: String?
+    var roomId: String!
     
     init(dictionary: NSDictionary) {
```

Comment:
You're right, [link](http://stackoverflow.com/questions/9970864/do-i-need-to-call-super-init-or-super-initwithcoder-etc-for-nsobject). Nvm then :)